webapp/src/ts/services/debug.service.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Manage debug mode.
+ *
+ * When enabled:
+ *
+ *   - persist setting to a cookie
+ *   - put PouchDB in debug mode
+ *
+ * Note: When debugging, make sure to enable the logs by selecting 'verbose' and 'info' options
+ * in the console of the browser's developer tools.
+ */
+
+import { Injectable } from '@angular/core';
+import { CookieService } from 'ngx-cookie-service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DebugService {
+  readonly cookieName = 'medic-webapp-debug'
+
+  constructor(private cookieService: CookieService) { }
+
+  get() {
+    return !!this.cookieService.get(this.cookieName);
+  }
+
+  set(bool:boolean) {
+    const db = window.PouchDB;
+
+    if (bool) {
+      db.debug.enable('*');
+      this.cookieService.set(this.cookieName, bool.toString(), 360, '/');
```

Comment:
Does this cookie do anything now? 

I think it should enable `debug` logs in production settings here: 
https://github.com/medic/cht-core/blob/4750-angular-10-take-2/webapp/src/ts/services/feedback.service.ts#L79

Reply:
Ohhh I'll check that, thanks

Reply:
Done. fixed & tested.